src/Framework/Bootstrap/AbstractSetupGacela.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gacela\Framework\Bootstrap;
+
+use Gacela\Framework\Config\GacelaConfigBuilder\ConfigBuilder;
+use Gacela\Framework\Config\GacelaConfigBuilder\MappingInterfacesBuilder;
+use Gacela\Framework\Config\GacelaConfigBuilder\SuffixTypesBuilder;
+
+abstract class AbstractSetupGacela implements SetupGacelaInterface
+{
+    /**
+     * Define different config sources.
+     */
+    public function buildConfig(ConfigBuilder $configBuilder): ConfigBuilder
+    {
+        return $configBuilder;
+    }
+
+    /**
+     * Define the mapping between interfaces and concretions, so Gacela services will auto-resolve them automatically.
+     *
+     * @param array<string,mixed> $externalServices
+     */
+    public function buildMappingInterfaces(MappingInterfacesBuilder $mappingInterfacesBuilder, array $externalServices): MappingInterfacesBuilder
+    {
+        return $mappingInterfacesBuilder;
+    }
+
+    /**
+     * Allow overriding gacela resolvable types.
+     */
+    public function buildSuffixTypes(SuffixTypesBuilder $suffixTypesBuilder): SuffixTypesBuilder
+    {
+        return $suffixTypesBuilder;
+    }
+
+    /**
+     * @return array<string,mixed>
+     */
+    public function externalServices(): array
+    {
+        return [];
+    }
+
+    /**
+     * @deprecated in favor of `externalServices()`
+     *
+     * @return array<string,mixed>
+     */
+    public function globalServices(): array
+    {
+        return $this->externalServices();
+    }
```

Comment:
This method was deprecated one month ago on version [0.16.0](https://github.com/gacela-project/gacela/blob/0.16.0/src/Framework/Setup/SetupGacela.php#L92).
Maybe we could drop it in this release @Chemaclass?

Reply:
Good idea. Feel free to do this in a follow up PR.